open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -34,7 +34,6 @@
     "* Kernel Density Estimation Plots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    def create_assert_df():
</pre><p>expected_result = pd.DataFrame(...) You don't need to create a function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
ok